Corona-Warn-App/src/main/res/layout/fragment_confirm_check_in.xml

```diff
@@ -206,6 +206,7 @@
                         android:background="?android:attr/listDivider" />
 
                     <androidx.constraintlayout.widget.ConstraintLayout
+                        android:id="@+id/confirm_checkin_settings_card_checkout_time_row"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:orientation="horizontal">
```

Comment:
it is not related to the fix. there is no `orientation`  for `ConstraintLayout`

Reply:
got rid of it